testdata/scripts/scope.txt

```diff
@@ -0,0 +1,46 @@
+go build main.go
```

Comment:
for the sake of avoiding more `go build` and `garble build` calls, which will make the tests slower, can you add all this to `syntax.txt`? That test is suited for all kinds of AST edge cases, so I think it fits nicely. You could add a separate file there to clarify the difference, such as `scopes.go`.